Countly.js

```diff
@@ -472,10 +445,10 @@ Countly.updateRemoteConfigExceptKeys = function(keyNames, callback){
 
 Countly.getRemoteConfigValueForKey = function(keyName, callback){
     CountlyReactNative.getRemoteConfigValueForKey([keyName.toString() || ""], (value) => {
```

Comment:
if `keyName` is not required, maybe we can define it as the default parameter and we have to check the type of some values such as String or Number in a separate function.

Reply:
In this case "keyName" is a mandatory parameter.

Reply:
OK. I see the function is `getRemoteConfigValueForKey` which passed `keyName` and that's a required parameter. In that case, we shouldn't use the logical `OR` operator in the params. I think in the JavaScript side, all mandatory parameters should be checked in a separate function, and empty parameters should be checked before being passed to the Native direction.

Reply:
Sounds like a reasonable improvement. But we'll add it in a future refactor